.changeset/poor-pillows-roll.md

```diff
@@ -0,0 +1,10 @@
+---
+'@oriflame/backstage-plugin-score-card': minor
+---
+
+Allow any entity to be scored.
+
+BREAKING CHANGES:
+
+- `systemEntityName` is replaced by `entityRef`
```

Comment:
Thanks for this. Maybe a minor improvement:
```suggestion
- `SystemScore` is renamed to `EntityScore` (and in a similar fashion all other `System*` components)
- `systemEntityName` in `EntityScore` is replaced by `entityRef`
```

Reply:
And I believe this is all! **Perfect work, thanks!** If you agree with the above (feel free to rephrase it), please commit it and I will approve the PR.